app/enterprise/1.3-x/kong-manager/administration/rbac/rbac.md

```diff
@@ -1,55 +1,3 @@
 ---
-title: RBAC in Kong Manager
-book: admin_gui
+redirect_to: enterprise/latest/kong-manager/administration/rbac
```

Comment:
(If it's possible, can it read `{{page.kong_version}}`? If not, the hardcoded version would do.)

Reply:
No it can't read `{{page.kong_version}}` unfortunately, I did `latest`as it seems like the type of thing that will be forgotten in releases and end up leading to the incorrect version aka maintenance hell but I agree it would be better if it linked to the same version.

Reply:
But by doing `latest`, we're _ensuring_ that it'll go to the incorrect place in future versions...

Reply:
So maybe our workaround is to check for versioned links?